transformations.py

```diff
@@ -0,0 +1,114 @@
+import pandas as pd
+import re
+import time
+
+
+class Field_Transformations:
+    def __init__(self, dataframe):
+        self.dataframe = dataframe
+        self.possition_abbr()
+        self.add_dept_codes()
+        self.parse_pay_location()
+        self.parse_work_location()
+        self.clean_address()
+        self.clean_homePhone()
+        self.clean_hireDate()
+
+    """
+    When paycom_job_title is formatted as "Academics Associate (0002/1005/ADASO)"", the following 
+    method extracts the capitalized, abbreviated form of the position (i.e. ADASO). Not all field values are
+    in this format so it first checks if "(" is included in the string
+    """
+
+    def possition_abbr(self):
+        new_column = []
+        for result in self.dataframe["paycom_job_title"]:
```

Comment:
For loops are considered an [anti-pattern](https://pandas.pydata.org/pandas-docs/stable/user_guide/basics.html#iteration) in terms of making updates to pandas dataframes. You generally want to use vectorization. You may want to use the [df.apply function](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.apply.html) here with a lambda expression.

You could redefine this as:
```python
def position_abbr(self, job_title):
    if "(" in job_title:
        value = job_title.split("/")[2]
        return value[:-1]
```
Then you can apply it to the dataframe like:

```python
self.dataframe.apply(lambda row: self.position_abbr(row.paycom_job_title), axis=1)
```